client/components/room/Chat.jsx

```diff
@@ -61,8 +60,8 @@ class Chat extends React.Component {
         roomName: location.pathname
       };
       this.props.socket.emit('chat message', messageObj);
-      // eventually, we will use the 'fromMe' property to tag messages
-      // as from the sender so that they can render differently on the page.
+      messageObj.fromMe = true;
```

Comment:
Now that we are storing the userName of each message, it would probably be more modular to use the username rather than adding in this extra field to keep track of.